GVFS/GVFS.Platform.Mac/MacPlatform.cs

```diff
@@ -1,33 +1,43 @@
-﻿using GVFS.Common;
+﻿using System.Runtime.InteropServices;
+using GVFS.Common;
 using GVFS.Common.FileSystem;
 using GVFS.Common.Tracing;
-using GVFS.Platform.POSIX;
-
+using GVFS.Platform.POSIX;
+
 namespace GVFS.Platform.Mac
 {
     public partial class MacPlatform : POSIXPlatform
     {
         public MacPlatform()
             : base(
-                installerExtension: ".dmg")
+                installerExtension: ".dmg",
+                supportsGVFSService: true)
         {
-        }
-
+        }
+
         public override IDiskLayoutUpgradeData DiskLayoutUpgrade { get; } = new MacDiskLayoutUpgradeData();
         public override IKernelDriver KernelDriver { get; } = new ProjFSKext();
 
         public override string GetOSVersionInformation()
         {
             ProcessResult result = ProcessHelper.Run("sw_vers", args: string.Empty, redirectOutput: true);
             return string.IsNullOrWhiteSpace(result.Output) ? result.Errors : result.Output;
-        }
-
-        public override FileBasedLock CreateFileBasedLock(
-            PhysicalFileSystem fileSystem,
-            ITracer tracer,
+        }
+
+        public override FileBasedLock CreateFileBasedLock(
+            PhysicalFileSystem fileSystem,
+            ITracer tracer,
             string lockPath)
         {
             return new MacFileBasedLock(fileSystem, tracer, lockPath);
-        }
-    }
+        }
+
+        public override string GetCurrentUser()
+        {
+            return Getuid().ToString();

```

Comment:
Can this go in POSIXPlatform rather than MacPlatform?  It [appears to be valid on Linux](http://man7.org/linux/man-pages/man2/getuid.2.html) as well.

cc: @chrisd8088 @kivikakk as FYI

Reply:
moved it to POSIX